no-browser-site-scrapers/Curative/index.js

```diff
@@ -16,12 +16,20 @@ module.exports = async function GetAvailableAppointments() {
                     body += chunk;
                 });
                 res.on("end", () => {
-                    response = JSON.parse(body);
+                    try {
+                        response = JSON.parse(body);
+                    } catch (e) {
+                        console.error("JSON.parse failed: " + e);
```

Comment:
can we do a `reject(e)` instead? and then just update line 11 to `(resolve, reject)` instead of just `(resolve)`? That way we stop executing instead of returning an empty string which could be miscronstrued as "no availability" or something.

Reply:
added `reject` to the mix